<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SessionOptionsGeneralWidget</class>
 <widget class="QWidget" name="SessionOptionsGeneralWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Protocol</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxProtocol">
       <item>
        <property name="text">
         <string notr="true">Telnet</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Serial</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Local Shell</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">Raw</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Named Pipe</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>233</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
